src/pages/RGBPP/TransactionList/List/index.tsx

```diff
@@ -0,0 +1,192 @@
+import { TFunction, useTranslation } from 'react-i18next'
+import styles from './styles.module.scss'
+import { TransactionLeapDirection } from '../../../../components/RGBPP/types'
+import AddressText from '../../../../components/AddressText'
+import { Link } from '../../../../components/Link'
+import { localeNumberString } from '../../../../utils/number'
+import SortButton from '../../../../components/SortButton'
+import FilterButton from '../../../../components/FilterButton'
+import { useStatistics } from '../../../../services/ExplorerService'
+import { useIsMobile } from '../../../../hooks'
+
+const RGBTransactionList: React.FC<{ list: Transaction[] }> = ({ list }) => {
+  const [t] = useTranslation()
+  const statistics = useStatistics()
+  const tipBlockNumber = parseInt(statistics?.tipBlockNumber ?? '0', 10)
+
+  const filterFields = getFilterList(t)
```

Comment:
Use `key` of i18n instead of its `value` as the index. `value` can be any charset and probably introduces an encoding problem.

Reply:
891b4111

Reply:
c5991a91